.travis.yml

```diff
@@ -1,23 +1,26 @@
 language: php
 
 cache:
-    directories:
-      - $HOME/.composer/cache/files
+  directories:
+    - $HOME/.composer/cache/files
 
 install:
-    - COMPOSER_MEMORY_LIMIT=-1 composer install --no-interaction --prefer-dist
+  - COMPOSER_MEMORY_LIMIT=-1 composer install --no-interaction --prefer-dist
 
 before_script:
-    - composer install
+  - composer install
 
 script:
-    - vendor/bin/phpunit
-    - COMPOSER_MEMORY_LIMIT=-1 composer phpstan-setup && composer phpstan
-allow_failures:
-    - php: 7.4snapshot
+  - vendor/bin/phpunit
+  - COMPOSER_MEMORY_LIMIT=-1 composer phpstan-setup && composer phpstan
 
 matrix:
-    include:
-        - php: 7.2
-        - php: 7.3
-        - php: 7.4snapshot
+  include:
+    - php: 7.2
+    - php: 7.3
+    - php: 7.4snapshot
+
+  fast_finish: true
+
+  allow_failures:
+    - php: 7.4snapshot
```

Comment:
Do we need this still?

Reply:
Yep, 7.4 is failing here :(
![Screenshot 2019-11-13 at 13 42 26](https://user-images.githubusercontent.com/880657/68764685-74b31b00-061b-11ea-9d12-10d2eb5aac97.png)